spec/models/package_manager/maven/maven_central_spec.rb

```diff
@@ -3,6 +3,20 @@
 require "rails_helper"
 
 describe PackageManager::Maven::MavenCentral do
+  describe ".latest_version" do
+    context "when maven-metadata.xml's version contained an interpolation string" do
+      let(:project_name) { "io.github.caffetteria:data-service-opencmis" }
+
+      it "scrapes the maven HTML to find a real version" do
+        latest_version = VCR.use_cassette("maven-central/data-service-opencmis") do
```

Comment:
Is it possible to include only the relevant bits of the yml in here, perhaps via a stubbed method, so it's easier to understand what the structure is that we're working with in this test?

Reply:
done!